src/components/DuosHeader.js

```diff
@@ -14,7 +114,8 @@
     this.state = {
       showSupportRequestModal: false,
       hover: false,
-      dacChairPath: '/chair_console'
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/DataBiosphere/duos-ui/pullRequest?prid=6963768)
```suggestion
      dacChairPath: "/chair_console",
```